sdk/identity/identity/test/msalTestUtils.ts

```diff
@@ -13,6 +13,13 @@ import { assert } from "chai";
 import { OperationTracingOptions, setSpan, context as otContext } from "@azure/core-tracing";
 import { SpanGraph, setTracer } from "@azure/test-utils";
 import { MsalBaseUtilities } from "../src/msal/utils";
+import { isNode } from "@azure/core-util";
+import * as dotenv from "dotenv";
+
+// Browser tests fail without this.
```

Comment:
I don't understand how it can be possible that browser tests failed without this.

Reply:
My bad, I meant the opposite 😅